specification/cognitiveservices/data-plane/ContentModerator/readme.nodejs.md

```diff
@@ -6,7 +6,7 @@ Please also specify `--node-sdks-folder=<path to root folder of your azure-sdk-f
 ``` yaml $(nodejs)
 nodejs:
   package-name: azure-cognitiveservices-contentmoderator
-  package-version: 1.0.0-preview
+  package-version: 3.0.0
   output-folder: $(node-sdks-folder)/lib/services/contentModerator
   override-client-name: ContentModeratorAPIClient
```

Comment:
Recommend to remove override-client-name (which means it will use ContentModeratorClient as client name) for consistency with other cognitiveservices SDKs.